tests/test_http.sh

```diff
@@ -24,8 +24,36 @@ rlJournalStart
         assert_up_and_running
     rlPhaseEnd
 
+    rlPhaseStartTest "Should not display SSL warning for HTTPS connection"
+        rlRun -t -c "docker exec -i kiwi_web /Kiwi/manage.py createsuperuser \
+            --username testadmin --email testadmin@domain.com --noinput"
+        rlRun -t -c "cat tests/set_testadmin_pass.py | docker exec -i kiwi_web /Kiwi/manage.py shell"
+        IP_ADDRESS=`docker inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' kiwi_web`
+        rm -f /tmp/testcookies.txt
+        rm -f /tmp/testdata.sslon
+        rlRun -t -c "curl -k -L -o- -c /tmp/testcookies.txt https://$IP_ADDRESS:8443/"
+        CSRF_TOKEN=`grep csrftoken /tmp/testcookies.txt | cut -f 7`
+        rlRun -t -c "curl -e https://$IP_ADDRESS:8443/accounts/login/ -d username=testadmin \
```

Comment:
see ";auto" for referrer. It could work and save you some typing.

Reply:
This seems to break HTTPS test. HTTP works fine.